app/models/school.rb

```diff
@@ -51,6 +51,10 @@ class School < ApplicationRecord
             presence: true,
             uniqueness: true
 
+  validates :induction_tutor_email,
+            notify_email: true,
+            allow_nil: true
+
```

Comment:
Maybe another validation to check for both fields to be present when one of them is present?